### PR TITLE
Initial work getting summary data / table for location page

### PR DESCRIPTION
**Pull request by pbeshai**
Sets up storing location info. The table does not yet include selected client ISPs nor has the table been fully populated.

image

### CODE REVIEW

**Comment by pbeshai** [comment]
ping @vlandham

File: src/components/SummaryTable/SummaryTable.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  static propTypes = {
+    data: PropTypes.array,
+  }
+
+  render() {
+    const { data } = this.props;
+
+    return (
+      <div className="SummaryTable">
+        <TacoTable columns={columns} data={data} />
```
Comment (vlandham) [inline src/components/SummaryTable/SummaryTable.jsx]: real professional code there.

File: src/containers/LocationPage/LocationPage.jsx [hunk 2]
```diff
@@ -452,11 +455,16 @@ class LocationPage extends PureComponent {
   }
 
   renderFixedSummaryData() {
+    const { summary = {} } = this.props;
```
Comment (vlandham) [inline src/containers/LocationPage/LocationPage.jsx]: is there a suggestion for when to do this vs having defaultProps?

**Comment by vlandham** [comment]
+1 🎉 :taco

File: src/containers/LocationPage/LocationPage.jsx (hunk 2 above)
Comment (pbeshai) [inline src/containers/LocationPage/LocationPage.jsx]: I think default props is better in general, but when I did this I was wondering if I wanted to handle not having summary data differently (e.g. return null) and so I did this as a temporary fix. I think this presently has the table headers rendered but I'll have to investigate more.

**Pull request merged**